pyproject.toml

```diff
@@ -45,9 +45,9 @@ pylint-flask = "0.6"
 requirements-detector = ">=1.2.0"
 PyYAML = "*"
 mccabe = "^0.7.0"
-flake8 = "<6.0.0"
-pyflakes = ">=2.2.0,<3.*"
-pycodestyle = ">=2.9.0"
+flake8 = ">=6.0.0"
+pyflakes = ">=3.1.0,<3.2.0"
```

Comment:
this, and `pycodestyle` below are lifted straight from the latest flake8 requirements, see their [conda recipe](https://github.com/conda-forge/flake8-feedstock/blob/main/recipe/meta.yaml) - since I wanted to always grab the latest flake8, this was in line with that; but I agree that if one is not immediately interested in the (current) latest `flake8`, a more lax approach is needed